hdk-rust/src/api.rs

```diff
@@ -53,37 +50,6 @@ lazy_static! {
   pub static ref AGENT_LATEST_HASH: &'static HashString = &GLOBALS.agent_latest_hash;
 }
 
-impl From<DNA_NAME> for JsonString {
-    fn from(dna_name: DNA_NAME) -> JsonString {
-        JsonString::from(RawString::from(dna_name.to_string()))
-    }
-}
-impl From<DNA_HASH> for JsonString {
-    fn from(dna_hash: DNA_HASH) -> JsonString {
-        JsonString::from(HashString::from(dna_hash.to_string()))
-    }
-}
-impl From<AGENT_ID_STR> for JsonString {
-    fn from(agent_id: AGENT_ID_STR) -> JsonString {
-        JsonString::from(RawString::from(agent_id.to_string()))
-    }
-}
-impl From<AGENT_ADDRESS> for JsonString {
-    fn from(agent_address: AGENT_ADDRESS) -> JsonString {
-        JsonString::from(Address::from(agent_address.to_string()))
-    }
-}
-impl From<AGENT_INITIAL_HASH> for JsonString {
-    fn from(agent_initial_hash: AGENT_INITIAL_HASH) -> JsonString {
-        JsonString::from(HashString::from(agent_initial_hash.to_string()))
-    }
-}
-impl From<AGENT_LATEST_HASH> for JsonString {
-    fn from(agent_latest_hash: AGENT_LATEST_HASH) -> JsonString {
-        JsonString::from(HashString::from(agent_latest_hash.to_string()))
-    }
-}
-
```

Comment:
Why did you remove these From implementations? This makes possible to write

```rust
hdk::link_entries(AGENT_ADDRESS, "post", post)
```

instead of

```rust
hdk::link_entries(JsonString::from(HashString::from(AGENT_ADDRESS.to_string())), "post", post)
```

Reply:
They looked pretty useless to me. Will update